### ISSUE TITLE
Not getting internet (I'm able to connect the client)

### ISSUE THREAD

**Issue by arif-b-khan** [vijayshinva/AzureWireGuard#4]
I used your script to deploy the wireguard servers. 
Everything got deployed successfully. I'm able to connect the client using client configuration file.
But I'm not getting any internet.

I tried many things to troubleshoot. 
Looked into Network Gateway, Disabled VM firewall etc
Nothing worked for me. I'm still not sure what is the issue.

**Comment by arif-b-khan** [comment]
Thanks for providing the script

**Comment by arunavo4** [comment]
Had the same issue, I had used wireguard before but can't remeber what i used to fix it, figured it was a DNS issue on my end. But I ended up using tailscale, where it can do a ton more stuff but with just the click of a button I can configure any client out of  the connected clients as an exit node for all my internet traffic make it so damn easy like any other off the market vpn. I think I might write a new client on top of wireguard and open source it. might be super useful. setting up wireguard can be a hassle. without proper docs.
